packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardUtils.ts

```diff
@@ -82,65 +82,44 @@ export const formatReleaseCardData = (
   }
 }
 
-const createButtonToExploreDataQueryString = (
+export const createButtonToExploreDataPathAndQueryString = (
   schema: ReleaseCardSchema,
   data: (string | null)[],
   buttonToExploreDataConfig?: ButtonToExplorePageConfig,
 ) => {
   if (!buttonToExploreDataConfig) return null
+  const path = getValueFromData(
+    schema,
+    data,
+    buttonToExploreDataConfig.sourcePathColumnName,
+  )
+  if (!path) {
+    console.warn(
+      `Column not found in source table or cell did not have value in source table for ${buttonToExploreDataConfig.sourcePathColumnName}`,
+    )
+    return null
+  }
```

Comment:
Output a warning if the user specified a `buttonToExploreDataConfig`, but a value for the path was not found in the source table. Assume that this is more likely to be an issue with the config or source table than by design. 

If we find that users only specify an explore button path for a subset of release cards by design, could change to a log message instead.